exercises/hamming/test/test_hamming.c

```diff
@@ -50,13 +70,17 @@ int main(void)
 {
    UnityBegin("hamming.c");
```

Comment:
Not a concern with this PR, but we should standardize on the name we use here. It's just an identifier for the test AFAIK, so maybe we just use the name instead of the filename.

```
>> grep -r -i --include test_* UnityBegin
exercises/acronym/test/test_acronym.c:   UnityBegin("test/test_acronym.c");
exercises/all-your-base/test/test_all_your_base.c:   UnityBegin("test/test_all_your_base.c");
exercises/allergies/test/test_allergies.c:   UnityBegin("test/test_allergies.c");
exercises/anagram/test/test_anagram.c:   UnityBegin("anagram.c");
exercises/atbash-cipher/test/test_atbash_cipher.c:   UnityBegin("test/test_atbash_cipher.c");
exercises/beer-song/test/test_beer_song.c:   UnityBegin("test/test_beer_song.c");
exercises/binary/test/test_binary.c:   UnityBegin("test/test_binary.c");
exercises/binary-search/test/test_binary_search.c:   UnityBegin("test/test_binary_search.c");
exercises/bob/test/test_bob.c:   UnityBegin("test/test_bob.c");
exercises/clock/test/test_clock.c:   UnityBegin("clock.c");
exercises/difference-of-squares/test/test_difference_of_squares.c:   UnityBegin("test/test_difference_of_squares.c");
exercises/gigasecond/test/test_gigasecond.c:   UnityBegin("test/test_gigasecond.c");
exercises/grains/test/test_grains.c:   UnityBegin("test/test_grains.c");
exercises/hamming/test/test_hamming.c:   UnityBegin("hamming.c");
exercises/hello-world/test/test_hello_world.c:   UnityBegin("test/test_hello_world.c");
exercises/isogram/test/test_isogram.c:   UnityBegin("test/test_isogram.c");
exercises/largest-series-product/test/test_largest_series_product.c:   UnityBegin("largest_series_product.c");
exercises/leap/test/test_leap.c:   UnityBegin("test/test_leap.c");
exercises/meetup/test/test_meetup.c:   UnityBegin("test/test_meetup.c");
exercises/nth-prime/test/test_nth_prime.c:   UnityBegin("test/test_nth_prime.c");
exercises/nucleotide-count/test/test_nucleotide_count.c:   UnityBegin("test/test_rna_transcription.c");
exercises/palindrome-products/test/test_palindrome_products.c:   UnityBegin("test/test_palindrome_products.c");
exercises/pangram/test/test_pangram.c:   UnityBegin("test/test_pangram.c");
exercises/pascals-triangle/test/test_pascals_triangle.c:   UnityBegin("test/test_pascals_triangle.c");
exercises/perfect-numbers/test/test_perfect_numbers.c:   UnityBegin("test/test_perfect_numbers.c");
exercises/phone-number/test/test_phone_number.c:   UnityBegin("test/test_phone_number.c");
exercises/raindrops/test/test_raindrops.c:   UnityBegin("raindrops.c");
exercises/react/test/test_react.c:   UnityBegin("test/test_react.c");
exercises/rna-transcription/test/test_rna_transcription.c:   UnityBegin("test/test_rna_transcription.c");
exercises/robot-simulator/test/test_robot_simulator.c:   UnityBegin("test/test_word_count.c");
exercises/roman-numerals/test/test_roman_numerals.c:   UnityBegin("test/test_roman_numerals.c");
exercises/scrabble-score/test/test_scrabble_score.c:   UnityBegin("test/test_scrabble_score.c");
exercises/series/test/test_series.c:   UnityBegin("test/test_word_count.c");
exercises/sieve/test/test_sieve.c:   UnityBegin("test/test_sum_of_multiples.c");
exercises/space-age/test/test_space_age.c:   UnityBegin("test/test_space_age.c");
exercises/sum-of-multiples/test/test_sum_of_multiples.c:   UnityBegin("test/test_sum_of_multiples.c");
exercises/triangle/test/test_triangle.c:   UnityBegin("test/test_triangle.c");
exercises/word-count/test/test_word_count.c:   UnityBegin("test/test_word_count.c");
```

Reply:
It looks like "test/<test_file_name>" is the most common pattern, should we standardize on that?